packages/components/stories/Enumeration.css

```diff
@@ -0,0 +1,3 @@
+.tc-enumeration button.inactive {
+  color: silver;
```

Comment:
Please keep the style in sass for consistency :).

Reply:
I can't make the style be applied if I rename to `Enumeration.scss`.
So I added the style to `preview-head.html` as it's done for other stories.